modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -102,64 +84,90 @@ public void testSendNotification() throws Exception {
 		notificationQueueEntryLocalService.deleteNotificationQueueEntry(
 			notificationQueueEntries.get(0));
 
-		notificationTemplate = _addNotificationTemplate(true);
+		// Multiples emails for each main recipients, comma separator
```

Comment:
@pedro-oliveira446 just:

```suggestion
		// Multiples emails for each main recipient, comma separator
```